### PR TITLE
bad wrong oops

### PR DESCRIPTION
**Pull request by RaveRadbury**
## About The Pull Request
Adding newscasters and cameras to pAI so they can make even more use of SS13's social features.

## Why It's Good For The Game
-Big Thought-
We're in a time of high population with many feeling unsatisfied with the available ghost roles. Adding more features to pAI's is a good way to ameliorate that. It's good for new and old players, especially the new ones as they can learn and participate in the culture of the game. Preserving the culture is important, right?

With decent ghost roles that are unlikely to cause incredible amounts of lag we might be able to argue for lifting the pop caps and having observers in. 

-Anyway-
This PR lets you make your pAI into your own personal publicist. Or combine these new abilities with the loudness booster and have a bard spreading word of your feats both through song and poem. These features don't get used enough and they are perfect for little gameboys that sit in PDA's. 

## Changelog
:cl:
add: pAI Newscaster
add: pAI Camera
/:cl:

This is a work in progress.

- [x] Add newscasters to pAI's
- [x] The newscaster is functional
- [x] Add cameras to pAI's
- [x] Update the pAI hud to include camera icons
- [x] Make sure camera works
- [ ] Integrate the two features into the software list
- [ ] Assign reasonable RAM prices

### CODE REVIEW

File: code/modules/mob/living/silicon/pai/pai.dm [hunk 1]
```diff
@@ -38,7 +38,8 @@
 	var/subscreen			// Which specific function of the main screen is being displayed
 
 	var/obj/item/pda/ai/pai/pda = null
-
+	var/obj/machinery/newscaster		// A newscaster
```
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/pai/pai.dm]: having a machine inside a mob. What could possibly go wrong.

Comment (RaveRadbury) [reply code/modules/mob/living/silicon/pai/pai.dm]: The eventual plan here is to have an entire space station within a pAI. I call it Project Turtles: All the Way Down

Comment (OverlyGeneric) [reply code/modules/mob/living/silicon/pai/pai.dm]: Are tiny fedoras included?

**Comment by OverlyGeneric** [comment]
Are tiny fedoras included with this program?

**Comment by RaveRadbury** [comment]
After this merges I'll have to play pAI as Mr. New Space Vegas. Let me just put on my newsman fedora...

**Comment by OverlyGeneric** [comment]
> After this merges I'll have to play pAI as Mr. New Space Vegas. Let me just put on my newsman fedora...

YIPPIE-YAY THERE'LL BE NO WEDDING BELLS TODAY.

File: code/modules/mob/living/silicon/pai/pai.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/silicon/pai/pai.dm]: *nervous laughing*

File: code/game/objects/objs.dm [hunk 2]
```diff
@@ -123,7 +123,7 @@
 			if ((M.client && M.machine == src))
 				is_in_use = TRUE
 				ui_interact(M)
-		if(isAI(usr) || iscyborg(usr) || IsAdminGhost(usr))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/objs.dm]: lol

**Comment by RaveRadbury** [comment]
Oh god, I pushed on to the wrong branch. Must fix!

**Comment by RaveRadbury** [comment]
What a horrible mess this is. I'm just going to scrap all this and try again. smh

**Pull request closed without merging**